### DISCUSSION TITLE
DDNS-GO能否调用KOOK的webhook

### DISCUSSION THREAD

**Discussion by gyh00** [jeessy2/ddns-go#956] [Q&A]
我在试图把ddns-go的webhook部署到kook上时遇到了困难，kook的webhook需要进行Challenge 验证，但是我对此并不了解，附上kook的说明
![image](https://github.com/jeessy2/ddns-go/assets/60257651/aa4d96b9-0676-4c5d-87a9-ec505a8e574b)
![image](https://github.com/jeessy2/ddns-go/assets/60257651/bc163525-f2b2-4ed7-b763-5eb4156751da)

**Comment by jeessy2** [comment]
这种应该不得行，换个简单的吧
